### DISCUSSION TITLE
"activeClassName" (imported from react-router-dom v6) is not working

### DISCUSSION THREAD

**Discussion by jaxkashif34** [tailwindlabs/tailwindcss#6729] [Help] [answered]
"activeClassName" (imported from react-router-dom v6) is not working is not working directly when applied classes to activeClassName but worked when use custom css or applied style in global css file

**Comment by sharjeelyunus** [comment, marked as answer]
NavLink activeClassName prop does not exists anymore
With V6 you have to manually do that and you can by passing a function to the className prop we have on React, like this

`<NavLink to={url} className={(navData) => navData.isActive ? 'm-2 mb-0 text-blue-700 border-b-2 dark:text-blue-300 border-blue-700 pb-2' : 'm-2 mb-0' }>{text}</NavLink>`

**Reply by femendesf** [reply]
Thankssssssss

**Comment by Soulaimane07** [comment]
Nice work broo
